2021-01-16 15:24:44,820 INFO    MainThread:2173095 [wandb_setup.py:_flush():70] setting env: {}
2021-01-16 15:24:44,821 INFO    MainThread:2173095 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-01-16 15:24:44,821 INFO    MainThread:2173095 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-16 15:24:44,821 INFO    MainThread:2173095 [wandb_init.py:_log_setup():318] Logging user logs to /home/whiteashes/Project/FedML/fedml_experiments/FedAvgExp/wandb/run-20210116_152444-nv9n9mdk/logs/debug.log
2021-01-16 15:24:44,821 INFO    MainThread:2173095 [wandb_init.py:_log_setup():319] Logging internal logs to /home/whiteashes/Project/FedML/fedml_experiments/FedAvgExp/wandb/run-20210116_152444-nv9n9mdk/logs/debug-internal.log
2021-01-16 15:24:46,715 INFO    MainThread:2173095 [wandb_run.py:_console_start():1308] atexit reg
2021-01-16 15:24:46,715 INFO    MainThread:2173095 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.REDIRECT
2021-01-16 15:24:46,715 INFO    MainThread:2173095 [wandb_run.py:_redirect():1178] Redirecting console.
2021-01-16 15:24:46,715 INFO    MainThread:2173095 [redirect.py:install():205] install start
2021-01-16 15:24:46,716 INFO    MainThread:2173095 [redirect.py:install():220] install stop
2021-01-16 15:24:46,716 INFO    MainThread:2173095 [redirect.py:install():205] install start
2021-01-16 15:24:46,716 INFO    MainThread:2173095 [redirect.py:install():220] install stop
2021-01-16 15:24:46,716 INFO    MainThread:2173095 [wandb_run.py:_redirect():1222] Redirects installed.
2021-01-16 15:24:46,717 INFO    MainThread:2173095 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 1
2021-01-16 15:24:46,717 INFO    MainThread:2173095 [wandb_run.py:_restore():1250] restore
2021-01-16 15:24:46,717 INFO    MainThread:2173095 [redirect.py:uninstall():224] uninstall start
2021-01-16 15:24:46,717 INFO    MainThread:2173095 [redirect.py:_stop():279] _stop: stdout
2021-01-16 15:24:46,717 INFO    MainThread:2173095 [redirect.py:_stop():285] _stop closed: stdout
2021-01-16 15:24:46,717 INFO    stdout    :2173095 [redirect.py:_pipe_relay():121] relay done saw last write: stdout
2021-01-16 15:24:46,717 INFO    stdout    :2173095 [redirect.py:_pipe_relay():137] relay done done: stdout
2021-01-16 15:24:46,718 INFO    MainThread:2173095 [redirect.py:_stop():291] _stop joined: stdout
2021-01-16 15:24:46,718 INFO    MainThread:2173095 [redirect.py:_stop():293] _stop rd closed: stdout
2021-01-16 15:24:46,718 INFO    MainThread:2173095 [redirect.py:uninstall():228] uninstall done
2021-01-16 15:24:46,718 INFO    MainThread:2173095 [redirect.py:uninstall():224] uninstall start
2021-01-16 15:24:46,718 INFO    MainThread:2173095 [redirect.py:_stop():279] _stop: stderr
2021-01-16 15:24:46,718 INFO    MainThread:2173095 [redirect.py:_stop():285] _stop closed: stderr
2021-01-16 15:24:46,718 INFO    stderr    :2173095 [redirect.py:_pipe_relay():121] relay done saw last write: stderr
2021-01-16 15:24:46,718 INFO    stderr    :2173095 [redirect.py:_pipe_relay():137] relay done done: stderr
2021-01-16 15:24:46,718 INFO    MainThread:2173095 [redirect.py:_stop():291] _stop joined: stderr
2021-01-16 15:24:46,718 INFO    MainThread:2173095 [redirect.py:_stop():293] _stop rd closed: stderr
2021-01-16 15:24:46,718 INFO    MainThread:2173095 [redirect.py:uninstall():228] uninstall done
2021-01-16 15:24:47,449 INFO    MainThread:2173095 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "b67b7344f07a4bf790948af99b4aab00"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 1104
      total_bytes: 1104
    }
  }
}

2021-01-16 15:24:49,453 INFO    MainThread:2173095 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "a294309adbe94059b905751b4252a0f0"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 3841
      total_bytes: 3841
    }
  }
}

2021-01-16 15:24:50,500 INFO    MainThread:2173095 [wandb_run.py:_show_files():1614] logging synced files
2021-01-16 15:24:50,501 INFO    MainThread:2173095 [internal.py:handle_exit():139] Internal process exited
